react-state/my-app/src/ToggleButton.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+
+export default function ToggleButton({ text, color }) {
+  const [isClicked, setIsClicked] = useState(false);
+  console.log('value returned by useState:', isClicked)
+
+  function handleClick() {
+    console.log('value before calling setter:', isClicked)
+    setIsClicked(!isClicked);
+    console.log('value after calling setter:', isClicked)
+  }
+
+  return (
+    <button onClick={handleClick} style={isClicked ? { backgroundColor: color, fontSize:20, fontWeight: "bold" } : { backgroundColor: "white" }}>
+      {text}
+    </button>
+  );
+}
+
+function ToggleLamberto() {
```

Comment:
Fun! See if you can get it to spin fast when the mouse is inside the logo (without clicking).